src/schemas/components/SchemaFormProperties.vue

```diff
@@ -65,13 +68,30 @@
   }
 
   function setValue(propertyKey: string, value: unknown): void {
-    emit('update:values', {
-      ...props.values,
-      [propertyKey]: value,
-    })
+    patches.push({ propertyKey, value })
+
+    flush()
   }
 
   function getRequired(propertyKey: string): boolean {
     return props.parent.required?.includes(propertyKey) ?? false
   }
+
+  const patches: { propertyKey: string, value: SchemaValue }[] = []
+
+  const flush = debounce(() => {
```

Comment:
This debounce helps collect multiple updates that happen at once and emits them together. Otherwise each emit overwrites the previous value because multiple emits can happen before the prop reflects the change.